snap/gui/boinc.desktop

```diff
@@ -0,0 +1,18 @@
+[Desktop Entry]
+Type=Application
+Version=1.0
```

Comment:
I believe, `set-client-version.py` script should update this file as well to include correct version of the BOINC

Reply:
And by default here should be the current master version (in this case 7.23.0)